### DISCUSSION TITLE
quality check after plot creation

### DISCUSSION THREAD

**Discussion by dimitrus** [Chia-Network/chia-blockchain#6163] [Ideas]
After creating a plot, I propose to add the ability to check the quality while the file is on ssd, and then transfer it to hdd. Checking for ssd takes a second, with 1000 it takes 20 seconds.
Add a variable to enable proof checking. 

**chia plots create --check-proofs 5..1000**

Time for phase 4 = 533.112 seconds. CPU (89.560%) Wed May 12 06:26:37 2021
Approximate working space used (without final file): 269.334 GiB
Final File size: 101.350 GiB
Total time = 17027.819 seconds. CPU (155.440%) Wed May 12 06:26:37 2021
**Proofs 40 / 30, 1.3333   f:\chia\plots\plot-k32-2021-05-12-01-42.plot**
Copied final file from "d:\\chia\\t1\\plot-k32-2021-05-12-01-42.plot.2.tmp" to "f:\\chia\\plots\\plot-k32-2021-05-12-01-42.plot.2.tmp"
Copy time = 649.030 seconds. CPU (7.210%) Wed May 12 06:37:26 2021
Removed temp2 file "d:\\chia\\t1\\plot-k32-2021-05-12-01-42.plot.2.tmp"? 1
Renamed final file from "f:\\chia\\plots\\plot-k32-2021-05-12-01-42.plot.2.tmp" to "f:\\chia\\plots\\plot-k32-2021-05-12-01-42.plot"

**Comment by tiny0little** [comment]
best to use ```Issues -> Make a Request / Request a new feature or idea here``` for that
